interface.html

```diff
@@ -24,5 +24,16 @@ <h2><small>Redirect after logging in</small></h2>
 
     <!-- Link action provider -->
     <div id="action"></div>
+    <div class="form-group col-sm-4 control-label">
+      <label>Do you want to allow users to show/hide their password when typing?</label>
+      <p class="help-block"><small>An icon will be shown in the password field, allowing users to toggle between masked characters and plain text.</small></p>
+      <div class="checkbox checkbox-icon password-show-checkbox">
+        <input type="checkbox" class="password-show-checkbox" id="password-show-checkbox">
```

Comment:
Avoid using the ID attribute. If it must be used, please append it with the widget ID.

```suggestion
        <input type="checkbox" class="password-show-checkbox" id="password-show-checkbox-{{id}}">
```